### ISSUE TITLE
Episode error

### ISSUE THREAD

**Issue by shizhuocaoa** [yanpanlau/DDPG-Keras-Torcs#51]
![torcs_erros](https://user-images.githubusercontent.com/44232865/47088088-40baca00-d250-11e8-83a2-64fefe1e2a5b.png)
When I tried run your file ddpg.py as your introduction, there are some problems. Just like what is showed in the picture, the Episode is zero all the time, and the car don't move at all. I just learn RL for a short time, so I don't understand why. Would you please help me? The is my we-chat number: wa7739977526. Thank you very much.

**Comment by BCWang93** [comment]
HI.Do you have this problem?when I run the ddpg.py,the terminal is output like this:
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Timeout for client answer
Do you know how to solution this problem?Thanks!

**Comment by lingyunli1994** [comment]
I see that the loss in your screenshot is always 0. Have you not changed the training mode, but have been running mode?

**Comment by BCWang93** [comment]
hello,Do you solved this problem?

**Comment by shushu-nantong** [comment]
@BCWang93 
The Timeout problem can be solved by checking the "snakeoil3_gym.py" file. If the snakeoil3_gym.py can be run without timeout errors, the ddpg.py can also work.

However, anyone knows how to solve the 0 episode problem?
